packages/TelegramClient-Core.package/TCCMessage.class/instance/sender.st

```diff
@@ -1,4 +1,4 @@
 accessing
 sender
 
-	^ sender
\ No newline at end of file
+	^ self core users getUserFor: self senderID
```

Comment:
Would making a separate method for this be a good idea to clarify intent (i.e. you might not be searching for this code in an accessor). On the other hand, creating another method in-between would also bloat the code base. Maybe this method could be renamed? That would violate "uniform access" though...

Just wanted to point this out, not sure if or what we should do about it